docs/application/dotnet/guides/internationalization/localization.md

```diff
@@ -0,0 +1,253 @@
+    //update your App UI
+});
+```
+
+
+<br><br>
+## Image Localization
+Tizen project supports localized images(resources) using different resource directories.
+Tizen uses res.xml file to specify information about the directory where localized resources(e.g. Image, Sound, and so on) are contained.
+The res.xml file is automatically generated when building your application.
+
+<br><br>
+**To add the localized resource directories in your application in Visual Studio:**
+
+- In Solution Explorer, Select Tizen project. And then, Go to Tools -> Tizen -> Resource Manager<br>
+![local_res_mgr_1](media/local_res_mgr_1.png)
+
+<br><br>
+- Resource Manager opens configurations -> Open the language drop-down list<br>
+![local_res_mgr_2](media/local_res_mgr_2.png)
+
+<br><br>
+- Select the language to add -> click Add button<br>
+![local_res_mgr_3](media/local_res_mgr_3.png)<br>
+![local_res_mgr_4](media/local_res_mgr_4.png)
+
+<br><br>
+- Resource directories are automatically created in Tizen project as follows.<br>
+  res.xml file will be generated in res directory of Tizen project after building an app.<br>
+![res.xml](media/local_res_xml.png)<br>
+![res.xml code](media/local_res_xml_code.png)
+
+<br><br>
+_Your application can sometimes run in a locale which you don't provide images for. In that case, Tizen loads the default image from the resource content directory (yourApp.Tizen/res/content/ ). If there is no default image within resource content directory and a device sets the locale which you don't provide images for, an error occurs._
+
+![default_image](media/local_res_default_image.png)
+
+<br><br>
+When you detect locale changes, you need to update the resource culture and then update the texts and images you want to provide for localization.
+As for localized images, you can make custom image renderer to load the proper locale-specific images.
+Tizen provides the path of locale-specific images via ResourceManager.TryGetPath/GetPath methods. With this, you can change 
+
+**class LocalizedImageRenderer :ImageRenderer**
+```cs
+        using TizenResourceManager = Tizen.Applications.ResourceManager;
+  
+        public LocalizedImageRenderer() : base()
+        {
+            SystemSettings.LocaleLanguageChanged += SystemSettings_LocaleLanguageChanged;
+        }
+ 
+        // Invoked every time the language setting has been changed
+        private void SystemSettings_LocaleLanguageChanged(object sender, LocaleLanguageChangedEventArgs e)
+        {
+            // Get the path of a proper image based on locale and update the source of an image
+            Element.Source = TizenResourceManager.TryGetPath(TizenResourceManager.Category.Image, fileName);
+        }
+```
+
+
+<br><br>
+## Application Name Localization
+You can add localized Application name and icon using `tizen-manifest.xml` in the Tizen project.
+
+- Open the `tizen-manifest.xml` and select the Localization tab and click Add button in Name.
```

Comment:
Open the `tizen-manifest.xml`, select the Localization tab and click **Add** to add Name.

Reply:
done